CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+ * Formatting with `xml.format.emptyElements`: `expand` chokes on malformed XML. See [lemminx#650](https://github.com/eclipse/lemminx/issues/650).
+ * Fix format problem with multiple root elements. See [#634](https://github.com/redhat-developer/vscode-xml/issues/634).
+ * Remove darwin-arm64 platform from generic publishing call.. See [#734](https://github.com/redhat-developer/vscode-xml/pull/734).
+ * Server binaries for "osx-x86_64" are not found. See [#729](https://github.com/redhat-developer/vscode-xml/issues/729).
+
+### Build
+
+ * Make sure that 'vsce' is installed in nodes that use it.. See [#789](https://github.com/redhat-developer/vscode-xml/pull/789).
+ * Fix problem matcher. See [#779](https://github.com/redhat-developer/vscode-xml/pull/779).
+ * Explicitly set 'main' as target branch for cloning of lemminx repo.. See [#750](https://github.com/redhat-developer/vscode-xml/pull/750).
+ * Update Jenkins build scripts for branch renaming. See [#747](https://github.com/redhat-developer/vscode-xml/pull/747).
+ * Enable dependabot. See [lemminx#1309](https://github.com/eclipse/lemminx/pull/1309).
+ * Unit tests should not store data in the .lemminx folder under the user's home directory.. See [lemminx#1265](https://github.com/eclipse/lemminx/issues/1265).
+ * Fix CI badge, add some more badges. See [lemminx#1262](https://github.com/eclipse/lemminx/pull/1262).
+ * Update Jenkins and GH Actions build scripts to reflect branch renaming. See [lemminx#1261](https://github.com/eclipse/lemminx/pull/1261).
+ * Set up linting for vscode-xml. See [#438](https://github.com/redhat-developer/vscode-xml/issues/438).
+
+### Other
+
+ * RelaxNG documentation. See [#787](https://github.com/redhat-developer/vscode-xml/pull/787).
+ * Update description for emptyElements and joinContentLines. See [#780](https://github.com/redhat-developer/vscode-xml/pull/780).
+ * Update description for preserveEmptyContent with experimental formatter. See [#778](https://github.com/redhat-developer/vscode-xml/pull/778).
+ * Update description for preservedNewLines with experimental formatter. See [#777](https://github.com/redhat-developer/vscode-xml/pull/777).
+ * Update description for joinCDATALines with experimental formatter. See [#776](https://github.com/redhat-developer/vscode-xml/pull/776).
+ * Fix typo with foldings settings. See [#771](https://github.com/redhat-developer/vscode-xml/pull/771).
+ * Installs badge. See [#767](https://github.com/redhat-developer/vscode-xml/pull/767).
+ * Update badges in README. See [#766](https://github.com/redhat-developer/vscode-xml/issues/766).
+ * Update description for closingBracketNewLine with experimental formatter. See [#762](https://github.com/redhat-developer/vscode-xml/pull/762).
+ * Update documentation to reflect that experimental formatter supports `xml.format.xsiSchemaLocationSplit`. See [#754](https://github.com/redhat-developer/vscode-xml/issues/754).
```

Comment:
Same for these. Mainly they're build related issues, which we can mention, but many are not as high value for someone reading them.

Reply:
removed